recipes/qt-advanced-docking-system/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set version = "4.0.1" %}
+
+package:
+  name: qt6-advanced-docking-system
+  version: {{ version }}
+
+source:
+  url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/archive/{{ version }}.tar.gz
+  sha256: c65357f630dc22b645b5dfa2684fb2748a347235267e952dc30986da9bf88f50
+  patches:
+    - 0001-CMakeLists.patch
+
+build:
+  number: 0
+  run_exports:
+    - qt6-advanced-docking-system
```

```suggestion
    - {{ pin_subpackage('qt6-advanced-docking-system', max_pin='x.x') }}
```

If you think you are stable to the major version, you could do:

```
    - {{ pin_subpackage('qt6-advanced-docking-system', max_pin='x') }}
```